vizro-core/docs/index.md

```diff
@@ -1,3 +1,285 @@
 # Vizro
 
 Vizro is a toolkit for creating modular data visualisation applications
+
+#### Quickstart
```

Comment:
Would rename this to `Tutorials` and then link to the overall tutorials page. Since this one is a special one (we likely want people to get to a specific tutorial), and will not contain too many sub-pages, we can think about linking the two tutorials also as a card.

Reply:
Or alternatively, for this section alone, we can actually link both tutorials. Yes I think that would be better. We should still have an explanation of the tutorial section

Reply:
Would keep the cards only for the main sections as pandas has it as well: https://panel.holoviz.org/getting_started/index.html

Also 4 cards make a nice grid 😄